### PR TITLE
fix: casbin example repo

### LINKED ISSUE #2514: [BUG] Casbin example not working
**Describe the bug**
Casbin is not working when flowing from the example.

**To Reproduce**
Just flowing from the document: https://refine.dev/docs/guides-and-concepts/access-control/#casbin
1. install casbin
2. write access control provider

The [node-casbin](https://github.com/casbin/node-casbin) is just for the backend, not the fronted. So there're errors when webpack building.

### PR DESCRIPTION
**Pull request by amit-ksh**
Fixed all the errors raised by webpack in Refine Casbin Example app.

Change I made - 

1) Upgraded the webpack to the latest version i.e. 5.0.1
     - **Reason**: webpack 4 does not support the nullish coalescing and optional chaining
1) Uses the react-app-rewired package for overriding the webpack config to support polyfills.
     - **Reason**: app created using `create-react-app` does not support the webpack config overriding and for that, we need [react-app-rewired](https://www.npmjs.com/package/react-app-rewired)
1) Added `.eslintignore` for ignore linting errors
1) Updated the `docs` accordingly

### Test plan (required)

Demonstrate the code is solid. If not, please add `WIP:` in its title.
[Working Demo](https://stackblitz.com/edit/pankod-refine-bmukwm)

### Closing issues
Closes #2514 

### Self Check before Merge

Please check all items below before review.

-   [x] Corresponding issues are created/updated or not needed
-   [x] Docs are updated/provided or not needed
-   [x] Examples are updated/provided or not needed
-   [x] TypeScript definitions are updated/provided or not needed
-   [x] Tests are updated/provided or not needed
-   [x] Changesets are provided or not needed

### CODE REVIEW

**Comment by omeraplak** [comment]
Hey @amit-ksh ,
Thank you for your contribution. Could you work `npm run lint:fix` on project root?

File: documentation/docs/advanced-tutorials/access-control.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ import accessControl from '@site/static/img/guides-and-concepts/access-control/a
 
 ## Introduction
 
-Access control is a broad topic where there are lots of advanced solutions that provide different set of features. **refine** is deliberately agnostic for its own API to be able to integrate different methods (RBAC, ABAC, ACL, etc.) and different libraries ([Casbin](https://casbin.org/), [CASL](https://casl.js.org/v5/en/), [Cerbos](https://cerbos.dev/), [AccessControl.js](https://onury.io/accesscontrol/)). `can` method would be the entry point for those solutions.
+Access control is a broad topic whsere there are lots of advanced solutions that provide different set of features. **refine** is deliberately agnostic for its own API to be able to integrate different methods (RBAC, ABAC, ACL, etc.) and different libraries ([Casbin](https://casbin.org/), [CASL](https://casl.js.org/v5/en/), [Cerbos](https://cerbos.dev/), [AccessControl.js](https://onury.io/accesscontrol/)). `can` method would be the entry point for those solutions.
```
Comment (aliemir) [inline documentation/docs/advanced-tutorials/access-control.md:13]:
```suggestion
Access control is a broad topic where there are lots of advanced solutions that provide different set of features. **refine** is deliberately agnostic for its own API to be able to integrate different methods (RBAC, ABAC, ACL, etc.) and different libraries ([Casbin](https://casbin.org/), [CASL](https://casl.js.org/v5/en/), [Cerbos](https://cerbos.dev/), [AccessControl.js](https://onury.io/accesscontrol/)). `can` method would be the entry point for those solutions.
```

Fixed the little typo 😅

**Comment by amit-ksh** [comment]
> Hey @amit-ksh , Thank you for your contribution. Could you work `npm run lint:fix` on project root?

Done.

File: examples/accessControl/casbin/src/index.tsx [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
+
```
Comment (omeraplak) [inline examples/accessControl/casbin/src/index.tsx:2]:
```suggestion
```

**Review by salihozdemir — APPROVED** [review]

**Review by omeraplak — APPROVED** [review]

**Pull request merged**